### DISCUSSION TITLE
How to import the antd CSSin v5.1?

### DISCUSSION THREAD

**Discussion by rmjuarez12** [ant-design/ant-design#39693] [General]
Hello all,

So, I am coming from v4.24.5 and I used to import the Antd CSS as:

`@import "antd/dist/antd.css";`

I am using SCSS for my stylesheets, and by just importing the above, it would import the Antd CSS file. When I updated, the above code throws an error and my app wont compile. If I remove it, it works, but now my app looks a bit messed up. I did looked at the module folder, and see that the file no longer exists in there. Is there a different way on how I should call that now?

**Comment by yoyo837** [comment]
`antd@5` is now a CSS-in-JS lib. See https://github.com/ant-design/ant-design/issues/38671.

**Reply by rmjuarez12** [reply]
Awesome, thank you so much! I should've read the changelog beforehand, I apologize.

**Reply by moodless3** [reply]
how did you fix it . am stuck here too . help pls

**Comment by arpitttttt** [comment]
CSS files are no longer included in package. Since CSS-in-JS supports importing on demand, the original antd/dist/antd.css has also been abandoned.
